test/message_scheduler_test.exs

```diff
@@ -0,0 +1,14 @@
+defmodule Elix.MessageSchedulerTest do
+  use ExUnit.Case
+  doctest Elix.MessageScheduler
+
+  test "sends a message" do
+    assert Elix.MessageScheduler.send_at(now(), :test) == :ok
+
+    # Actually testing this module will be a challenge...
```

Comment:
Also, `Elix.MessageScheduler` assumes the recipient will be a `GenServer`, which the test process is not. So I'd perhaps need a simpler GenServer forwarder process to receive messages and forward them to the test process.